src/components/Inputs.js

```diff
@@ -18,23 +31,24 @@ const Inputs = () => {
           value={title}
           type="text"
           onInput={(e) => setTitle(e.target.value)}
+          required
         />
         <input
           className="input title-input"
           placeholder="Author"
           value={author}
           type="text"
           onInput={(e) => setAuthor(e.target.value)}
+          required
         />
```

Comment:
- [OPTIONAL] I can see that you added required attribute  to your input tag, for for some reason, it is not working. I can still create empty books. Kindly give a closer look and fix it.

### Your Page
![your-page](https://user-images.githubusercontent.com/26597432/215256269-c45c62ce-32bf-4f1a-baba-66bfdc6e9d79.png)

Reply:
Thank you @Qoosim for this comment.I am going to resolve it.